gbpservice/nfp/config_orchestrator/agent/vpn.py

```diff
@@ -66,13 +66,9 @@ def _get_vpn_context(self, context, tenant_id, vpnservice_id,
                 'ipsecpolicies': ipsecpolicies,
                 'ipsec_site_conns': ipsec_site_conns}
 
-    def _get_core_context(self, context, tenant_id):
-        filters = {'tenant_id': [tenant_id]}
-        core_context_dict = common.get_core_context(context,
-                                                    filters,
-                                                    self._conf.host)
-        del core_context_dict['ports']
-        return core_context_dict
+    def _get_core_context(self, context):
```

Comment:
yes ,suresh you  are correct,we have to change there first .i am not sure there we can change or not ,need to look at that.with present code this is correct